<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="QNA-CD071" failures="0" tests="8" name="com.nxstudio.testCases.TC_Login_001" time="70.257" errors="1" timestamp="2023-06-17T18:27:37 IST" skipped="0">
  <testcase classname="com.nxstudio.testCases.TC_Login_001" name="launchBrowser" time="5.354"/>
  <system-out/>
  <testcase classname="com.nxstudio.testCases.TC_Login_001" name="verifyTitle" time="0.000">
    <error message="
Cannot inject @Test annotated Method [verifyTitle] with [interface org.testng.ITestResult].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [verifyTitle] with [interface org.testng.ITestResult].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:439)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:353)
at org.testng.internal.Parameters.createParameters(Parameters.java:708)
at org.testng.internal.Parameters.handleParameters(Parameters.java:884)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- verifyTitle -->
  <system-out/>
  <testcase classname="com.nxstudio.testCases.TC_Login_001" name="blankFields" time="10.628"/>
  <system-out/>
  <testcase classname="com.nxstudio.testCases.TC_Login_001" name="passwordBlankFields" time="10.544"/>
  <system-out/>
  <testcase classname="com.nxstudio.testCases.TC_Login_001" name="emailBlankFields" time="10.253"/>
  <system-out/>
  <testcase classname="com.nxstudio.testCases.TC_Login_001" name="invalidCredentials" time="9.909"/>
  <system-out/>
  <testcase classname="com.nxstudio.testCases.TC_Login_001" name="invalidEmailFormat" time="11.727"/>
  <system-out/>
  <testcase classname="com.nxstudio.testCases.TC_Login_001" name="Login" time="11.842"/>
  <system-out/>
</testsuite> <!-- com.nxstudio.testCases.TC_Login_001 -->
